contracts/Oracle/UniswapPairOracle.sol

```diff
@@ -13,6 +13,7 @@ import "./ICryptoPairOracle.sol";
 contract UniswapPairOracle is Ownable, ICryptoPairOracle {
     using FixedPoint for *;
 
+    uint8 private constant DECIMALS = 18;
```

Comment:
I had to set decimals for UniswapPairOracle because it also implements ICryptoPairOracle